DearPyGui/dearpygui/core.pyi

```diff
@@ -1002,6 +1002,10 @@ def set_individual_color(item: str, id: int, color: List[float]) -> None:
 	"""Sets an color of a theme item."""
 	...
 
+def hide_headers(table: str, hide: bool) -> None:
+	"""Hide a tables headers."""
+	...
+
```

Comment:
Did you manually put this in? Just an FYI, this file is automatically generated if you run _DearSandbox_ in release! This would also put it in alphabetical order.

Reply:
Delete the change.